### PR TITLE
WT-10137 Create a feature flag to enable/disable stats in page header

### PR DESCRIPTION
**Pull request by GWlodarek**

### CODE REVIEW

**Review by mm-ng — APPROVED** [review]
LGTM

File: src/support/global.c [hunk 1]
```diff
@@ -127,6 +127,7 @@ __global_once(void)
     /* Run-time configuration. */
 #ifdef WT_STANDALONE_BUILD
     __wt_process.fast_truncate_2022 = true;
+    __wt_process.page_stats_2022 = true;
```
Comment (bvpvamsikrishna) [inline src/support/global.c:130]: I think it is better if it starts with "false" till some testing is done in our local environments.

Comment (GWlodarek) [reply src/support/global.c]: Good idea, done.

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

**Pull request merged**